portfolio/src/main/java/com/google/sps/data/Login.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.data;
+
+/**
+ * Stores login status of the user and login/logout url.
+ */
+
+public class Login {
+
+    private boolean loggedIn;
+    private String url;
+
+    public Login(boolean loggedIn, String url) {
```

Comment:
A couple of initial thoughts: 
1) The two constructors have duplicate logic, which assigns to the loggedIn field. One constructor should call the other.
2) Would it be more useful to use the Url (uri?) java class? That way, "null" can be more meaningful and can be passed in as the default paramter.